integration-tests/src/test/java/oracle/weblogic/kubernetes/ItConfigDistributionStrategy.java

```diff
@@ -825,9 +825,11 @@ private void verifyResourceJDBC1Override(boolean configUpdated) {
     for (int i = 1; i <= replicaCount; i++) {
       appURI = "dsTest=true&dsName=" + dsName1 + "&" + "serverName=" + managedServerNameBase + i;
       String dsConnectionPoolTestUrl = baseUri + appURI;
-      HttpResponse<String> response = assertDoesNotThrow(() -> OracleHttpClient.get(dsConnectionPoolTestUrl, true));
-      assertEquals(200, response.statusCode(), "Status code not equals to 200");
-      assertTrue(response.body().contains("Connection successful"), "Didn't get Connection successful");
+      testUntil(() -> {
+        HttpResponse<String> response = assertDoesNotThrow(() -> OracleHttpClient.get(dsConnectionPoolTestUrl, true));
+        logger.info("Http response status code {0} \n Http response body {1} ", response.statusCode(), response.body());
+        return response.statusCode() == 200 && response.body().contains("Connection successful");
+      }, logger, "Waiting for http response code 200 and message Connection successful");
```

Comment:
You don't need to add the "Waiting for" here. It is already pre-added in testUntil() method.

Reply:
Fixed